sdk/__tests__/utils.ts

```diff
@@ -253,6 +269,38 @@ export async function transferWithChecks(
   );
 }
 
+async function waitForRelay(
+  msgId: WormholeMessageId,
+  dst: Ctx,
+  retryTime: number = 2000
+) {
+  console.log("Sleeping for 1 min to allow signing of VAA");
+  await new Promise((resolve) => setTimeout(resolve, 60 * 1000));
```

Comment:
Would like to find a way to set the consistency level for the relayer to instant so this does not need to wait for a full minute before checking